scripts/test/isis_powder/ISISPowderFocusCropTest.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import (absolute_import, division, print_function)
+import mantid.simpleapi as mantid
+from isis_powder.routines import focus
+import unittest
+
+
+class ISISPowderFocusCropTest(unittest.TestCase):
+    def test_crop_before(self):
+        x = [0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100]
+        y = [0, 0, 10, 30, 2000, 80, 50, 40, 30, 25, 30]
+        e = [5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5]
```

Comment:
Is the `e` array needed here? The `DataE` property of `CreateWorkspace` is optional and the errors are not tested. Same applies to the other tests below.